documentation/src/main/markdown/packagemgrs/python.md

```diff
@@ -99,4 +100,17 @@ Poetry detector attempts to run on your project if either a poetry.lock or pypro
 The Poetry detector parses poetry.lock for dependency information. If the detector discovers a pyproject.toml file but not a poetry.lock file, it will prompt the user to generate a poetry.lock by running `poetry install` and then run [company_name] [solution_name] again.
 The Poetry detector extracts the project's name and version from the pyproject.toml file.  If it does not find a pyproject.toml file, it will defer to values derived by git, from the project's directory, or defaults.
 
-When the `--detect.poetry.dependency.groups.excluded` property is specified, presence of both poetry.lock and pyproject.toml files is required for this detector to run successfully.
\ No newline at end of file
+When the `--detect.poetry.dependency.groups.excluded` property is specified, presence of both poetry.lock and pyproject.toml files is required for this detector to run successfully.
+
+## Setuptools detector
+
+Setuptools detector attempts to run on your project, after potential execution of other detectors, if a pyproject.toml file containing a build section with `requires = ["setuptools"]` or equivalent line is located. Setuptools scans can be run in both build, if a pip installation is available, and buildless mode if not.
```

Comment:
I know this is what we talked about but I did have to shuffle the order to make things work. If Setuptools finds the things you mention it will run instead of the other pip detectors. If it doesn't find them, or it finds them and fails, Detect will then try to run the pip detectors

Reply:
Ok, I will switch the order of detectors on the page. Thanks.